storage/plugins/book-club/src/PollController.php

```
@@ -472,6 +476,25 @@ public function closeExpiredPolls(): void
+     * list/read paths: club page poll list, mobile club detail and the
+     * dashboards. Only touches polls with status='open' AND closes_at in
+     * the past, so the close side effects (tally, winner transition,
+     * bookclub.poll.closed hook) run exactly once.
+     */
+    public function closeExpiredForClub(int $clubId): void
+    {
+        foreach ($this->repo->expiredOpenPollsForClub($clubId) as $poll) {
+            try {
+                $this->resolvePoll($poll, true);
+            } catch (\Throwable $e) {
+                SecureLogger::error('[BookClub] lazy close failed for poll ' . $poll['id'] . ': ' . $e->getMessage());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'closeExpiredForClub'. (undefined)

(StaticAccess)